src/commands/edge/channels.ts

```diff
@@ -28,13 +39,26 @@ export default class ChannelsCommand extends EdgeCommand {
 	static flags = {
 		...EdgeCommand.flags,
 		...outputListing.flags,
+		'include-read-only': flags.boolean({
```

Comment:
I don't have strong opinions either way, but we could choose to abstract the "read-only" terminology from users and just have the option be `--include-subscribed` or something.

Reply:
I would rather match the API unless it's particularly egregious so users looking at the API docs won't be too confused. I wish we could do aliases for arguments like we can for commands. I also _think_ there is some reason they avoided the term "subscribed"